recipes/Cm/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cm" %}
+{% set version = "9.0.0" %}
+{% set versions = version.split('.') %}
+
+package:
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+   url: http://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+   sha256: ffcd65f5ff779db07bc2fbe982da6277ffa495aa6d66300c3194bd4a344311a4
+
+build:
+  number: 0
+  skip: True  # [win or osx]
```

Comment:
Once the feedstock is created, I can help configure builds for these platforms, presuming the package builds on them in the first place.

Reply:
Unfortunately, the package doesn't build on OSX and windows as you can see from the checks [here](https://github.com/conda-forge/staged-recipes/runs/1878807711), due to combability issues on OSX and required libraries not available on Windows. But the developer is currently attempting to port the code to OSX and possibly windows.